tests/components/toolbar_test.js

```diff
@@ -154,6 +156,16 @@ describe("Toolbar Component", function() {
 
         expect(current).to.be.equal("header-two");
       });
+
+      it("triggers custom action", function() {
+        const items = this.wrapper.find(ToolbarItem);
+        const customItem = items.at(5);
+        const button = customItem.find("button");
+
+        button.simulate("click");
+
+        expect(this.actions[5].action).to.have.been.called;
```

Comment:
hi @gaku-sei, thanks.

Could you please ensure that editorState is passed as param?